lib/rmt/file_validator.rb

```diff
@@ -0,0 +1,48 @@
+require 'fileutils'
+
+class RMT::FileValidator
```

Comment:
Make it a mix-in and mix it into `RMT::Mirror`, and then make `deep_verify` an attribute of `RMT::Mirror` class? Just so that you don't have to pass `deep_verify` as a parameter everywhere.

Same probably could apply for `RMT::Deduplicator` and force copy/don't track options.

Reply:
Ok, I'll come up with a separated commit, maybe we can even delegate logging to `RMT::Deduplicator`, and eliminate the wrapper method `RMT::Mirror#deduplicate`, which only logs when a file is deduplicated.

Reply:
I implemented the mixins in c417d92. I guess it fits your idea.